expansion/src/main/java/no/unit/nva/expansion/model/IndexDocumentWrapperLinkedData.java

```diff
@@ -113,4 +114,8 @@ private Stream<String> fetchContentRecursively(URI uri) {
     private Optional<String> fetch(URI externalReference) {
         return uriRetriever.getRawContent(externalReference, APPLICATION_JSON_LD.toString());
     }
+
+    private Optional<String> fetchOrganizations(URI externalReference) {
+        return uriRetriever.getRawContent(externalReference, APPLICATION_JSON_LD.toString() + CRISTIN_VERSION);
```

Comment:
do you need a `; `between json ld and version? can you write a test asserting accept header is correct? Why is the caller method called a name containing recursively? does it do recursion here or is it just a name?

Reply:
"Why is the caller method called a name containing recursively? does it do recursion here or is it just a name?" No idea, but if you look closely it is recursive.

Reply:
"do you need a ; between json ld and version? can you write a test asserting accept header is correct?" Will try to write test for this.

Reply:
just wondering